td-shim-tools/src/bin/td-shim-layout-builder/main.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
+use clap::{Parser, ValueEnum};
+use td_shim_tools::layout_builder::{
+    self, parse_csv, parse_json, parse_layout_config, MemoryRegions,
+};
+
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
+pub enum ConfigType {
+    /// csv format config
+    CSV,
```

Comment:
JSON is enough. Why we need CSV?

Reply:
CSV format is optional.  Json is also supported.
Memory layout is more like to a table. CSV is designed for tables.
To a same layout CSV format 14 lines are needed, but Json format needs 54 lines. It's more clearer.

Reply:
Why we have to support both formats?

I think JSON is enough. Please remove CSV format at all.

Reply:
done